.eslintrc

```diff
@@ -44,7 +44,8 @@
         "no-param-reassign": 0,
         "react/no-did-update-set-state": 0,
         "react/no-did-mount-set-state": 0,
-        "no-unused-vars": [1, { "args": "none" }],
+        "no-unused-vars": 0,
+        "@typescript-eslint/no-unused-vars": [1, { "args": "none" }],
```

Comment:
It is from typescript eslint rule details - https://typescript-eslint.io/rules/no-unused-vars/#rule-details